### PR TITLE
Change pre-commit hook

### PR DESCRIPTION
**Pull request by BrunaNayara**
The `make setup` installed `pre-commit` to create and maintain git hooks. But running it resulted in this error 
```pre-commit install
make: pre-commit: Command not found
Makefile:2: recipe for target 'setup' failed
make: *** [setup] Error 127
```
I created a small pre-commit hook, and added black to requirements, because the only hook was black*.

\* Refer to the [CONTRIBUTING](https://github.com/okfn-brasil/diario-oficial/blob/main/CONTRIBUTING.md#automated-code-formatting) and [pre commit config](https://github.com/okfn-brasil/diario-oficial/blob/main/.pre-commit-config.yaml) files

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+black==19.10b0
```
Comment (sergiomario) [inline requirements.txt:1]: requirements.txt exists here: https://github.com/okfn-brasil/diario-oficial/blob/main/processing/requirements.txt

Comment (BrunaNayara) [reply requirements.txt:1]: Oh, nice. I'll point to that one :)

Comment (BrunaNayara) [reply requirements.txt:1]: Actually, this requirements is for the docker container and they are likely to conflict if installing directly out of docker.
So I don't think it's good installing them in the setup. They should only be installed in the docker steps. 
@sergiomario what do you think?

Comment (sergiomario) [reply requirements.txt:1]: Right. I understood the conflict, I'll do the merge like this.

**Comment by jvanz** [comment]
To keep the tradition of cool gif when a new contributor send the first PR:
image

**Comment by jvanz** [comment]
giphy

**Pull request merged**